modules/ocf/facts.d/le-cert-info

```diff
@@ -0,0 +1,30 @@
+#!/usr/bin/env python3
+import json
+import os
+from datetime import datetime
+
+from cryptography import x509
+from cryptography.hazmat.backends import default_backend
+
+LETSENCRYPT_CERT_DIR = '/var/lib/lets-encrypt/certs/'
+
+cert_data = {}
+
+for title in os.listdir(LETSENCRYPT_CERT_DIR):
+    try:
+        with open(os.path.join(LETSENCRYPT_CERT_DIR, title, 'cert.pem'), 'rb') as pem_file:
+            pem_data = pem_file.read()
+    except FileNotFoundError:
+        continue
+    except PermissionError as e:
+        print('try running this command as root or ocfletsencrypt')
+        raise e
+
+    cert = x509.load_pem_x509_certificate(pem_data, default_backend())
+    san = cert.extensions.get_extension_for_oid(x509.oid.ExtensionOID.SUBJECT_ALTERNATIVE_NAME)
+    cert_data[title] = {
```

Comment:
Is it possible that there could be multiple entries here with the same title? That could cause problems here.

Reply:
No. `title` comes from directory names, which must be unique within the /var/lib/lets-encrypt/certs/ directory